applications/nutanix-ai/2.4.0/helmreleases/nutanix-ai.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: source.toolkit.fluxcd.io/v1
 kind: OCIRepository
 metadata:
-  name: nutanix-ai
+  name: nutanix-ai-2.4.0-chart
```

Comment:
do we need `${releaseName}` on these too?

Reply:
its not needed but doesn't hurt ig. fixed in [e1a1b06](https://github.com/nutanix-cloud-native/nkp-nutanix-product-catalog/pull/89/commits/e1a1b06d65a56c9ec3fe36e4077cde242087571f) so that we are consistent.